Calibration/HcalAlCaRecoProducers/plugins/AlCaHcalHBHEMuonProducer.cc

```diff
@@ -190,10 +194,13 @@ AlCaHcalHBHEMuonProducer::AlCaHcalHBHEMuonProducer(const edm::ParameterSet& iCon
                                << labelEBRecHit_ << " EE " << labelEERecHit_ << " HBHE " << labelHBHERecHit_ << " MU "
                                << labelMuon_;
 
-  tok_ddrec_ = esConsumes<HcalDDDRecConstants, HcalRecNumberingRecord, edm::Transition::BeginRun>();
-  tok_htopo_ = esConsumes<HcalTopology, HcalRecNumberingRecord, edm::Transition::BeginRun>();
-  tok_respcorr_ = esConsumes<HcalRespCorrs, HcalRespCorrsRcd, edm::Transition::BeginRun>();
-  tok_geom_ = esConsumes<CaloGeometry, CaloGeometryRecord, edm::Transition::BeginRun>();
+  tok_ddrec0_ = esConsumes<HcalDDDRecConstants, HcalRecNumberingRecord, edm::Transition::BeginRun>();
+  tok_ddrec1_ = esConsumes<HcalDDDRecConstants, HcalRecNumberingRecord>();
+  tok_respcorr0_ = esConsumes<HcalRespCorrs, HcalRespCorrsRcd, edm::Transition::BeginRun>();
+  tok_respcorr1_ = esConsumes<HcalRespCorrs, HcalRespCorrsRcd>();
+  tok_geom0_ = esConsumes<CaloGeometry, CaloGeometryRecord, edm::Transition::BeginRun>();
+  tok_geom1_ = esConsumes<CaloGeometry, CaloGeometryRecord>();
+  tok_htopo_ = esConsumes<HcalTopology, HcalRecNumberingRecord>();
   tok_magField_ = esConsumes<MagneticField, IdealMagneticFieldRecord>();
   tok_chan_ = esConsumes<EcalChannelStatus, EcalChannelStatusRcd>();
   tok_sevlv_ = esConsumes<EcalSeverityLevelAlgo, EcalSeverityLevelAlgoRcd>();
```

Comment:
and all these should be moved to the constructor here
https://github.com/cms-sw/cmssw/blob/f7f3c5a38e71e6835a7bf89aa531aa694a629d6f/Calibration/HcalAlCaRecoProducers/plugins/AlCaHcalHBHEMuonProducer.cc#L157